package.json

```
@@ -23,15 +23,15 @@
     "lit": "^2.8.0"
   },
   "peerDependencies": {
-    "@alaskaairux/design-tokens": "^3.15.5",
     "@alaskaairux/icons": "^4.20.0",
-    "@aurodesignsystem/webcorestylesheets": "^4.8.0"
+    "@aurodesignsystem/design-tokens": "^4.1.1",
```

Per recent discussions on the Auro team we need to keep around the legacy versions as well for the time being.
```suggestion
    "@alaskaairux/design-tokens": "^3.15.5",
    "@aurodesignsystem/design-tokens": "^4.1.1",
```